_translator-files/po/de/Installation-for-Linux.po

```diff
@@ -42,7 +43,7 @@ msgstr "Installation auf Linux"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:9
 msgid "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
-msgstr "{% include breadcrumb.html root=\"Jamulus Nutzen\" branch1=\"Erste Schritte\" branch1-url=\"Getting-Started\" %}"
+msgstr "{% include breadcrumb.html root=\"Jamulus verwenden \" branch1=\"Setup - Erste Schritte mit Jamulus“ branch1-url=\"Getting-Started\" %}"
```

Comment:
The second quotation mark here `"Setup - Erste Schritte mit Jamulus“` is still wrong.

Reply:
How do you find this?
Is there a tool?

Corrected and committed again.

Thanks for all the guidance & patience!

Reply:
> How do you find this?
> Is there a tool?

I run jekyll on a clone of your branch and look at the errors.

> Thanks for all the guidance & patience!

No problemo! I'm just sorry about the learning curve. Believe me it's been simplified, but it's still not easy, especially if you need to learn to use GitHub.

Reply:
Agree ;-). Probably worth extending the (po4a) documentation?